flink/src/main/java/org/apache/iceberg/flink/sink/IcebergStreamWriter.java

```diff
@@ -57,6 +64,9 @@ public void open() {
 
     // Initialize the task writer.
     this.writer = taskWriterFactory.create();
+
+    // If we don't enable checkpoint, we will use processingTimeSerice to do commit,
+    boolean isCheckpointEnabled = getRuntimeContext().isCheckpointingEnabled();
```

Comment:
This appears to be unused (it's neither stored as a class field or used elsewhere in this constructor. Perhaps this is a bug or simply a left over artifact from some refactoring?

Reply:
Thanks Kyle, this was a remaining code I miss to clean before last refactor.

Reply:
Hi Kyle, I miss the code below and I have recommit it.